Sources/DeepLearning/Layer.swift

```diff
@@ -558,7 +557,7 @@ public struct TransposedConv2D: Layer {
         self.activation = activation
         self.strides = strides
         self.padding = padding
-        self.paddingIndex = padding == .same ? 0 : 1
+        self.paddingIndex = padding == .same ? 0: 1
```

Comment:
Seems like an accidental change :)
```suggestion
        self.paddingIndex = padding == .same ? 0 : 1
```